src/scripts/sites/accountStatement.js

```diff
@@ -64,9 +64,61 @@ chrome.storage.sync.get
                 });
                 
                 $formatDecimals();
+                $add_timespan_always();
             }
         }
-        
+
+        function $add_timespan_always() {
+
+            const always_a = document.createElement('a');
+            always_a.innerText = localization('timespan');
+            always_a.href = 'javascript:;';
+            always_a.setAttribute('data-value', 'always');
+
+            always_a.addEventListener('click', function (event) {
+                document.querySelectorAll('li.m-quickfilter-item > a:not([data-value = "always"])').forEach(function (elem) {
+                    elem.classList.remove('active')
+                });
+                always_a.classList.add('active');
+                document.querySelector('#period-from').value = '1.1.1950';
+                var d = new Date();
+                document.querySelector('#period-to').value = d.getDate()
+                    + '.' + d.getMonth()
+                    + '.' + d.getFullYear();
+                document.querySelector('#filter-button').click()
+            });
+
+            const always_li = document.createElement('li');
+            always_li.classList.add('m-quickfilter-item');
+            always_li.appendChild(always_a);
+            document.querySelector('#quickfilters').appendChild(always_li);
+
+            document.querySelectorAll('li.m-quickfilter-item > a:not([data-value = "always"])').forEach(function (elem) {
+                elem.addEventListener('click', function (event) {
+                    always_a.classList.remove('active');
+                })
+            });
+        }
+
+        function localization (field)
+        {
+            var translations =
+                {
+                    'timespan' :
+                        {
+                            'en' : 'All the time',
+                            'de' : 'Ganze Zeit',
+                            'pl' : 'Od początku',
+                            'cs' : '?',
```

Comment:
CZ: Kompletní historie

Reply:
Thank you, but you would need to translate all the messages in all the other files for this to work. And you're more than welcome to :). See [Limitations](https://github.com/DeeNaxic/mintos-extension#limitations) in the Readme.

Reply:
Ok, done :)

Reply:
I've just found your pull request with the translation. Let's leave it to @DeeNaxic to decide which PR goes first and which one has to be updated. Cheers.

Reply:
The PR looked great. I've tested it and merged it into the development branch. I have also added the translations as suggested by jpadera in the comments. This will be out with version 1.4 this week.